supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,30 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[command-line-commands-that-require-elevated-privileges]]
+== Command-line commands that require elevated privileges
+
+Some command-line interface (CLI) commands require that a user run them with "elevated privileges".  
```

Comment:
```suggestion
Some command-line interface (CLI) commands require that a user run them with elevated privileges. 
```
Avoid using quotes. It comes off like "air quotes". If you need to emphasize something, use italics. In this particular case, I see no reason to emphasize these words.

After reading the next sentence, I think you can combine it with this one, or just delete this one. This sentence is redundant, taking into account the heading.

Reply:
Deleted the first sentence. You're right. It's not needed.